Mpdfin/Mpd/Request.cs

```diff
@@ -146,4 +146,13 @@ public static Range ParseRange(U8String input)
 
         return result ? new(start, end) : throw new FormatException($"Invalid range {input}");
     }
+
+    bool IUtf8SpanFormattable.TryFormat(
+        Span<byte> utf8Destination,
+        out int bytesWritten,
+        ReadOnlySpan<char> format,
+        IFormatProvider? provider)
+    {
+        throw new NotImplementedException();
+    }
```

Comment:
I'm assuming this implementation is never called, but why? Is it only in the interface for situations when `ToU8String` is not implemented?